inc/saitypes.h

```diff
@@ -691,6 +691,40 @@ typedef struct _sai_segment_list_t
     sai_ip6_t *list;
 } sai_segment_list_t;
 
+/**
+ * @brief Defines a lane with its eye values
+ */
+typedef struct _sai_port_lane_eye_values_t
+{
+    uint32_t lane_number;
```

Comment:
would "lane" be enough ? you can add comment to that filed that thie is lane number, but single "lane" in this context is probably self explaining

Reply:
I'm fine with just lane.

Reply:
what is the unit for these for left/right/up/down values?